scripts/malloc-init.pl

```diff
@@ -1,70 +0,0 @@
-#!/usr/bin/perl
```

Comment:
Unfortunately our docs repository is not full yet, and a quick google search only showed the github links, so I think we are safe here